Source/RageCommunity.Library/Wrappers/NativeFireWrappers.cs

```diff
@@ -6,6 +6,15 @@ namespace RageCommunity.Library.Wrappers
 {
     public static partial class NativeWrappers
     {
-        
+        /// <summary>
+        /// Gets the position of the closes fire based on the given search position.
+        /// </summary>
+        /// <returns>
+        /// Returns true when a fire was found, otherwise false.
+        /// </returns>
+        public static Boolean GetClosestFirePosition(out Vector3 foundFirePosition, Vector3 searchPosition)
+        {
+            return NativeFunction.Natives.x352A9F6BCF90081F<Boolean>(out foundFirePosition, searchPosition.X, searchPosition.Y, searchPosition.Z); 
```

Comment:
You don't have to specify the X, Y, and Z coordinates.  You can use a normal Vector3
```suggestion
            return NativeFunction.Natives.x352A9F6BCF90081F<Boolean>(out foundFirePosition, searchPosition); 
```

Reply:
Where do you got this information? Native DB shows that this native has 4 parameters. Our, X, Y, Z

Reply:
I believe it's some Rage is able to do internally.  If you look at the `DrawMarker` wrapper, I do it with the position.  Feel free to test it out.